code-studio/build-js.js

```diff
@@ -31,6 +36,16 @@ build_commands.execute([
     commonFile: 'code-studio-common',
     shouldMinify: commander.min,
     shouldWatch: commander.watch
+  }),
+  build_commands.browserify({
+    srcPath: SRC_PATH + 'initApp/',
+    buildPath: BUILD_PATH,
+    filenames: [
+      'initApp.js'
+    ],
+    commonFile: 'initApp-common.js',
```

Comment:
I don't, and they seem to load just fine...

This might change if we stick other stuff (like leveltype_widget) in here.